src/libs/map_settings.c

```diff
@@ -154,6 +176,24 @@ void gui_init(dt_lib_module_t *self)
 
   gtk_box_pack_start(GTK_BOX(self->widget), GTK_WIDGET(hbox), TRUE, TRUE, 0);
 
+  if(!dt_conf_key_exists("plugins/map/epsilonfactor"))
```

Comment:
I suppose here we better have an entry in darktableconfig.xml.in. The type float should have the default value 0.25 and most importantly a min / max to ensure proper recovery when a value is corrupt. This is recent work, see for example ce34dc336895d937aaa7e4afe6f070e3e65e1895. And then you don't need to check for existence for this key.

Reply:
Is darktableconfig.xml.in the only file to set up (including for min max) ?

Reply:
no, but I don't find it ... I remember I've already got difficulty yo find it ...

Reply:
It is here : `data/darktableconfig.xml.in`

Reply:
I've run the build.sh script and I got it.
Is there a place (a way) to define the number of digits and the increment in config (it is .001 currently) ?
![image](https://user-images.githubusercontent.com/23012047/98482459-55c07800-21e0-11eb-8bae-48646dea607b.png)

Reply:
Well it looks like you are using the generated for from the template. The file to modify is committed and so not generated.

No there is no way to describe the increment. This must be done in dt.

Reply:
> No there is no way to describe the increment. This must be done in dt.

Where ? in gui/preferences.c ?
I remember I had to code something to wrap folder configuration for lut3d but I've forgotten where.

Reply:
For preferences I don't think we have a way to set the increment.

Reply:
Thanks. I've moved to integer to avoid the problem.